kubernetes/reporter/src/main/java/org/arquillian/cube/kubernetes/reporter/KubernetesReporterExtension.java

```diff
@@ -0,0 +1,15 @@
+package org.arquillian.cube.kubernetes.reporter;
+
+import org.jboss.arquillian.core.spi.LoadableExtension;
+
+
+public class KubernetesReporterExtension implements LoadableExtension{
+
+    @Override
+    public void register(ExtensionBuilder builder) {
+        final boolean reportedInClasspath = Validate.classExists("org.arquillian.recorder.reporter.ReporterExtension");
```

Comment:
Do you mean `reporterInClasspath`? maybe there is a typo as you used `reported`.